src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+* {
+  box-sizing: border-box;
+}
+
+.page {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  color: $main-text-color;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  border: 1px solid #F3F3F3;
+  border-radius: 5px;
+  background: url(../images/imac.jpeg) no-repeat;
+  background-size: 160px 134px;
+  background-position: top 32px center;
+  display: flex;
+  flex-direction: column;
+  padding: 206px 16px 17px;
+  width: 200px;
+
+  &__name {
+    color: $main-text-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    color: $secondary-text-color;
+    font-size: 10px;
+    font-weight: 400px;
+    line-height: 14px;
+    margin-bottom: 16px;
+    text-align: left;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: end;
+    margin-bottom: 24px;
+  }
+
+  &__review {
+    text-decoration: none;
+    color: $main-text-color;
+    font-size: 10px;
+    font-weight: 400;
+    display: inline-block;
+    line-height: 14px;
+
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-title {
+    color: $secondary-text-color;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price-amount {
+    color: $main-text-color;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__buttom {
+    background-color: $accent-color;
+    color: #FFF;
+    padding: 12px 70px;
+    border-radius: 5px;
+    text-transform: uppercase;
+    font-size: 14px;
+    text-align: center;
+    font-weight: 700;
+    line-height: 16px;
+    text-decoration: none;
+    height: 40px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. use padding instead